commands/cluster-links.md

```diff
@@ -0,0 +1,49 @@
+Each node in a Redis Cluster maintains a pair of long-lived TCP links with each
+peer in the cluster, one established toward the peer, another accepted from the
+peer.
+
+`CLUSTER LINKS` outputs information of all such peer links as an array, where each
+array element is a RESP3 map that contains various attributes and their values of
+an individual link.
```

Comment:
Maps aren't a native data type in RESP2, which is why it shows up as an array below, so perhaps just say:
```suggestion
`CLUSTER LINKS` outputs information of all such peer links as an array, where each array element is a map that contains attributes and their values for an individual link.
```